content.md

```diff
@@ -236,3 +237,29 @@ Tap on add photo button and select the image to display information that needs t
 ### 6.6 Tap the *CLEAR CACHE* button
 ![screenshot](./images/6/6.jpg)
 <div class="pagebreak"></div>
+
+## 7 How to back-up data to google drive
+
+### 7.1 Start from map view and tap the more icon at top-right of the screen
+![screenshot](./images/map.jpg)
+<div class="pagebreak"></div>
+
+### 7.2 Tap menu icon in the top-right of the screen then select *Saved Forms*
+![screenshot](./images/7/2.jpg)
+<div class="pagebreak"></div>
+
+### 7.3 Tap *BACKUP* in the top-right of the screen
+![screenshot](./images/7/3.jpg)
+<div class="pagebreak"></div>
+
+### 7.4 Tap *Save to drive* in the pop-up menu on the bottom of the screen
+![screenshot](./images/7/4.jpg)
+<div class="pagebreak"></div>
+
+### 7.5 Give a *Document title*, select an *Account* (Gmail) and *Folder* and select save in the bottom-right of the pop-up menu.
+![screenshot](./images/7/5.jpg)
+<div class="pagebreak"></div>
+
+### 7.6 If you have successfully uploaded data message at the bottom of the screen should display *Your X file is being updoaded to: My Drive*
```

Comment:
Rather just say that a message should display informing you of the successful upload to the data.